packages/x-tree-view-pro/README.md

```diff
@@ -15,7 +15,7 @@ This component has the following peer dependencies that you will need to install
 
 ```json
 "peerDependencies": {
-  "@mui/material": "^5.15.14",
+  "@mui/material": "^5.15.0",
```

Comment:
This was an intentional fix from me that I noticed only when checking this PR. 🙈 
The README is out of sync with the actual version of the peer dep in `package.json`. 😉 
https://github.com/mui/mui-x/blob/3782074a1fa2be00a57ded5e4f95a305878429c7/packages/x-tree-view-pro/package.json#L61

Reply:
Ah ok, I will add the change again